src/Symfony/Bundle/SecurityBundle/Resources/config/collectors.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\DependencyInjection\Loader\Configurator;
+
+use Symfony\Bundle\SecurityBundle\DataCollector\SecurityDataCollector;
+
+return static function (ContainerConfigurator $container) {
+    $container->services()
+        ->set('data_collector.security', SecurityDataCollector::class)
+        ->args([
```

Comment:
args and tag should be indented deeper than set

Reply:
Done